src/main/resources/assets/bongo/lang/en_us.json

```diff
@@ -15,6 +15,7 @@
   "bongo.team.green": "Team Green",
   "bongo.team.red": "Team Red",
   "bongo.team.black": "Team Black",
+  "bongo.tooltip.required": "bongo",
```

Comment:
This should be something like "Bingo Item". It's still a bingo mod. Just the modname is bongo. Also the ingame command is called bingo for that reason.